### PR TITLE
fixed UAVCAN MAG calibration application after reboot

### PR DESCRIPTION
**Pull request by buraksaruhan**
Please use [PX4 Discuss](http://discuss.px4.io/) or [Slack](http://slack.px4.io/) to align on pull requests if necessary. You can then open draft pull requests to get early feedback.

**Describe problem solved by the proposed pull request**
fixed UAVCAN MAG calibration application after reboot

**Test data / coverage**
Tested with px4 firmware 1.9.0 stable, made these changes, enabled UAVCAN_BUS with Qground, disabled other mag's, calibrated, rebooted. Calibration was still there. 

**Describe your preferred solution**
A clear and concise description of what you have implemented.

**Describe possible alternatives**
A clear and concise description of alternative solutions or features you've considered.

**Additional context**
Add any other related context or media.

### CODE REVIEW

**Comment by buraksaruhan** [comment]
Tested with px4 firmware 1.9.0 stable, made these changes, enabled UAVCAN_BUS with Qground, disabled other mag's, calibrated, rebooted. Calibration was still there.

**Comment by dagar** [comment]
This is because of the timing at boot?

**Comment by dagar** [comment]
Build failure.

image

**Comment by dagar** [comment]
Timing wise this is probably still worth backporting for v1.9.3.

**Comment by buraksaruhan** [comment]
Hang on, tried this on 1.9.0. Will try on 1.9.3

**Comment by dagar** [comment]
> Hang on, tried this on 1.9.0. Will try on 1.9.3

I'm saying once merged to master I'll backport the fix to v1.9 and tag a new release (v1.9.3).

**Comment by buraksaruhan** [comment]
Okey, thank you. :) 

Deleted the print's, builded just fine on my machine.

**Comment by buraksaruhan** [comment]
I made the change, do I need to do any more for my pull request to be accepted? Sorry, I'm not a frequent commiter, but I hope I'll be from now on!

File: src/modules/sensors/voted_sensors_update.cpp [hunk 1]
```diff
@@ -278,6 +278,7 @@ void VotedSensorsUpdate::parameters_update()
 
 				} else {
 					/* apply new scaling and offsets */
+					/* show mscale values */
```
Comment (mhkabir) [inline src/modules/sensors/voted_sensors_update.cpp]:
```suggestion
```

File: src/modules/sensors/voted_sensors_update.cpp [hunk 2]
```diff
@@ -764,6 +765,9 @@ void VotedSensorsUpdate::mag_poll(vehicle_magnetometer_s &magnetometer)
 				int32_t priority = 0;
 				orb_priority(_mag.subscription[uorb_index], &priority);
 				_mag.priority[uorb_index] = (uint8_t)priority;
+
+				//Force Scales & Offsets update first time we get data
```
Comment (mhkabir) [inline src/modules/sensors/voted_sensors_update.cpp]:
```suggestion
				/* force a scale and offset update the first time we get data */
```

**Pull request merged**